src/components/cc-block-detail/cc-block-detail.js

```diff
@@ -0,0 +1,114 @@
+import { css, html, LitElement } from 'lit';
+import { accessibilityStyles } from '../../styles/accessibility.js';
+import { iconRemixArrowUpSLine as iconArrowUp } from '../../assets/cc-remix.icons.js';
+import '../cc-icon/cc-icon.js';
+
+/**
+ * A display component with mostly HTML+CSS and an open/close toggle feature organized with slots to display information such as CLI commands.
+ *
+ * @cssdisplay block
+ *
+ * @slot tab-title - The title of the tab. Try to only use text.
+ * @slot link - A zone dedicated to link, for example to documentation.
+ * @slot content - A zone dedicated to main content.
+ */
+
+export class CcBlockDetail extends LitElement {
+  static get properties() {
+    return {
+      isOpen: { type: Boolean, reflect: true, attribute: 'is-open' },
+    };
+  }
+
+  constructor() {
+    super();
+
+    /** @type {Boolean} Sets the state of the toggle behaviour. */
+    this.isOpen = false;
+  }
+
+  _onClickToggle() {
+    this.isOpen = !this.isOpen;
+  }
+
+  render() {
+    return html`
+      <div class="menu">
+        <button class="tab ${this.isOpen ? 'open' : 'close'}" @click=${this._onClickToggle}>
```

Comment:
note: not a real issue but since the component has the `is-open` attribute reflected, we can remove the `${this.isOpen ? 'open' : 'close'}` (see below for CSS).

Reply:
I was thinking the same :+1: